index.markdown

```diff
@@ -4,28 +4,65 @@ title: Digital Tools for Trade Unions
-    url: '/joining-tool'
 ---
 
-## Join Together helps unions get more members&nbsp;by:
+## What makes Join Together so special?
 
-- Turning effective workplace organising and great campaigns into more members by making it quick and easy for people to join
-- Giving prospective members a great first impression of joining a union
-- Reducing member attrition and increasing revenue, by making it straightforward for people to update their contact information, payment details and their subs rate
-- Being designed and built for unions, fully hosted, secure, scalable and continually optimised to get better and better over time
-- Being built by a team who get unions
+Put simply, we are **100% focused on building the gold standard online joining experience for unions**. Every union has its own unique needs and requirements. We work with you and your team to tailor the perfect online joining experience for you and your members.
 
-[![The Communication Workers Union]({{ site.url }}{{ site.baseurl }}/assets/images/cwu-logo.svg)](https://www.cwu.org) 
-[![Bakers Food and Allied Workers Union]({{ site.url }}{{ site.baseurl }}/assets/images/bfawu-logo.png)](https://www.bfawu.org) 
-[![Writers Guild of Great Britain]({{ site.url }}{{ site.baseurl }}/assets/images/wggb-logo.jpg)](https://writersguild.org.uk) 
-[![Forsa the Irish Trade Union]({{ site.url }}{{ site.baseurl }}/assets/images/forsa-logo.png)](https://www.forsa.ie) 
-[![Trade Union Congress]({{ site.url }}{{ site.baseurl }}/assets/images/tuc-logo.jpg)](https://www.tuc.org.uk) 
+## Online join that is optimised to get you more members
+
+We sweat the details so that your online join gets you more members: 60% of applicants that start a Join Together join form go on to submit their application. How many extra members would your union get with a form that converted that well?
+
+## Bespoke forms that handle all the complexities of joining your union
+
+Whether it’s employer/sector-specific questions, complex rate calculation rules, deduction-at-source, or Direct Debit, the Join Together platform adapts to meet all the specific joining needs of your union.
+
+## Seamless integration with your existing CRM and processes
+
+Join Together lives in harmony alongside your existing membership system, and is able to automatically feed new applications directly into your CRM ready for your membership team to process. Including Salesforce, Dynamics365, APT Stratum, Wild Apricot and more.
+
+## Works alongside your website
+
+The Join Together system can be fully customised to fit with your existing brand and website, guaranteeing a smooth user experience for visitors joining your union. From your logo to brand colours, fonts and more.
+
+## Built to work beautifully on mobile and desktop
+
+With as much as 50% of applicants using a mobile device, you really cannot afford to offer a sub-standard experience to mobile users. Join Together’s forms work perfectly across all devices.
```

Comment:
```suggestion
With as many as 50% of applicants using a mobile device, you cannot afford to offer a substandard experience to mobile users. Join Together’s forms work perfectly across all devices.
```

Are we sure it's an "up to" scenario? I thought it was more than half – around 59% from mobile devices now? Or am I reading US-only stats? No need to hyphenate "substandard", it's in common use without.